server/routes/user.py

```diff
@@ -87,12 +89,19 @@ def index():
 def upload_import():
     # TODO: changeg SECRETE_PROJECT to APP_PROJECT
     # Upload import files to GCS
-    user_id = session["google_id"]
+    oauth_user_id = session["oauth_user_id"]
+    user_id = libutil.hash_id(oauth_user_id)
     project = current_app.config['SECRET_PROJECT']
     bucket_name = get_gcs_bucket(project)
     gcs_client = storage.Client(project=project)
-    bucket = gcs_client.get_bucket(bucket_name)
+    try:
+        bucket = gcs_client.get_bucket(bucket_name)
+    except exceptions.NotFound:
+        return f'Bucket {bucket_name} is not found', 500
+
     import_name = request.form.get('importName')
+    if import_name == '':
```

Comment:
`if not import_name` ?  (resilient to it being `None` too)

Reply:
Done